hack/update-dependencies-script/run.sh

```
@@ -1,11 +1,29 @@
+# This script will run all possible dependency upgrades
+# These are all of the subdirectories that include a run.sh script
 #!/bin/bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)